checkov/sca_package_2/runner.py

```diff
@@ -149,5 +149,14 @@ def upload_package_files(
                     package_files_to_persist.append(FileToPersist(file, os.path.relpath(file, root_folder)))
 
         logging.info(f"{len(package_files_to_persist)} sca package files found.")
+        if package_files_to_persist and bc_integration.bc_source and not bc_integration.bc_source.upload_results:
```

Comment:
What happens is `package_files_to_persist` is not empty, `bc_integration.bc_source` exists and `bc_integration.bc_source.upload_results` is True? We don't upload the package files? Something seems off in this predicate

Reply:
if upload_results is True, we dont to do anything since we call setup_bridgecrew_credentials function before we run any runner and it will initialize the s3 path and s3 client. and only if upload_results if False and there are package files to scan we will call setup_bridgecrew_credentials function to initialize the s3 path and s3 client for the first time.